runtime/common/orderedmap/orderedmap.go

```diff
@@ -108,23 +121,39 @@ func (om *KeyTypeValueTypeOrderedMap) Delete(key KeyType) (oldValue ValueType, p
 }
 
 // Len returns the length of the ordered map.
-func (om *KeyTypeValueTypeOrderedMap) Len() int {
+func (om OrderedMap[K, V]) Len() int {
+	if om.pairs == nil {
+		return 0
+	}
+
```

Comment:
I think builtin `len()` function already handles `nil`. But no harm having it here I guess